docs/mkdocs-user-docs.yml

```diff
@@ -10,19 +10,25 @@ plugins:
       j2_variable_end_string: "]]-->"
 markdown_extensions:
   - admonition
+  - attr_list
+  - footnotes
```

Comment:
Nitpick: I believe there are no more footnotes on these pages, so this _could_ be dropped?

```suggestion
```